node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,10 @@
+import { readFile } from 'node:fs/promises';
+
+const contentPath = `./${process.argv[2]}`
```

Comment:
See above comments about ./ and error logging. Apply those here and in read-dijkstra.js.